src/styles/style.chatitem.ts

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components';
+
+export const MyMessageWrapper = styled.div`
+    display : flex;
+`
+export const MySendTime = styled.span`
+    font-size : 0.3rem;
+    margin-top : auto;
+    margin-right : 3px;
+    margin-bottom : 20px;

```

Comment:
```suggestion
    margin: auto 3px 20px 0;
```
css짱짱맨 성준님은 아시겠지만.. margin, padding 은 하나하나 속성을 지정할 수도, 한번에 쓸 수도 있습니다!
저는 margin, padding 등의 속성이 2개를 넘어가면 축약하는 걸 선호하는 편입니다 ㅎㅎ